src/test/java/com/amazon/opendistroforelasticsearch/security/auditlog/impl/TracingTests.java

```diff
@@ -332,11 +333,17 @@ public void uncaughtException(Thread t, Throwable e) {
     @Test
     public void testAdvancedMapping() throws Exception {
         Settings settings = Settings.builder()
-                .put(ConfigConstants.OPENDISTRO_SECURITY_COMPLIANCE_HISTORY_READ_WATCHED_FIELDS, "*")
```

Comment:
Is the change necessary? Can't it rely on `settings` to configure audit in the test not directly related to it?